capa/main.py

```diff
@@ -415,6 +434,20 @@ def handle_common_args(args):
     # disable vivisect-related logging, it's verbose and not relevant for capa users
     set_vivisect_log_level(logging.CRITICAL)
 
+    # logger = logging.getLogger()
+
+    # # use [/] after the logger name to reset any styling,
+    # # and prevent the color from carrying over to the message
+    # LOGFORMAT = "[dim]%(name)s[/]: %(message)s"
+
+    # if logger.hasHandlers():
+    #      logger.handlers.clear()
```

Comment:
can you explain why you think these need to be cleared?

Reply:
I used
```python
if logger.hasHandlers():
    logger.handlers.clear()
```
because of duplicate logging message as a result of multiple handlers which resulted in the logging message being printed twice.

The reason being —
- `logging.basicConfig(level=logging.<LEVEL>)` creates a default `StreamHandler` which is added to the root logger.
-  `logger.add_handler(rich_handler)` added `RichHandler` for the `capa` logger which is a child of the root logger thus propagating to root logger.

I fixed these issues in 33249f5beb43efabdde0aad712cfbffda6233c5a by removing the `logging.basicConfig` and configuring the root logger to use `RichHandler`

Reply:
great! As you've found, we're no longer in the "use the simple basic config setup" so removing those calls is a good solution. Thanks!